astropy/utils/data.py

```diff
@@ -17,7 +17,9 @@
 import socket
 import sys
 import time
-import urllib
+import urllib.request
```

Comment:
Will probably conflict with #6658.

Reply:
Hopefully a merge will handle it gracefully, but we'll see.

Reply:
I'll fix it if/when the first round of CI passes.

Reply:
CI looks okay. It probably doesn't make sense to restart the one that failed with an URLError if you rebase the PR (which runs the whole tests again anyway).